croncat/src/grpc.rs

```diff
@@ -63,15 +63,15 @@ impl GrpcSigner {
         let out = self
             .client
             .query_client
-            .query_contract(&self.client.cfg.contract_address, msg)
+            .query_contract(&self.client.cfg.contract_address.to_owned().unwrap(), msg)
```

Comment:
How would you feel to make new struct like `VerifiedChainConfig`, so we have less to `unwrap()`. 

And I think here is redundant clone?
```suggestion
            .query_contract(self.client.cfg.contract_address.as_ref().unwrap(), msg)
```

Reply:
Missed that those were being passed by reference! Fixing that now. 

Also that's a possibility with the struct, let me think about that.